src/features/Instructors/AddInstructors/index.jsx

```diff
@@ -99,12 +101,41 @@ const AddInstructors = ({ isOpen, onClose }) => {
     return null;
   };
 
+  const handleEditInstructor = async (e) => {
+    e.preventDefault();
+    try {
+      const formData = new FormData();
+      formData.append('enrollment_privilege', formState.instructor.hasEnrollmentPrivilege);
```

Comment:
Just a question but why at the moment to edit we need to send `enrollment_privilege` instead of `has_enrollment_privilege`?

Reply:
Backend miscommunication, someone made get service and other patch service and name it differnt.